### PR TITLE
Distributed mnist example fails to run

### PR DESCRIPTION
**Pull request by johnugeorge**
With the latest pytorch, Dataloader argument batch size has to be an integer.
Also, usage of indexing 0-dim Tensor is deprecated.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pytorch-operator/36)

### CODE REVIEW

**Comment by johnugeorge** [comment]
@jose5918  can you add lgtm label too?
